.env.example

```
@@ -0,0 +1,14 @@
+SITE_URL=https://example.com
+SITE_AUTHOR=Your Name
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SITE_AUTHOR key should go before the SITE_URL key

(UnorderedKey)